homework/java/shelest_vladimir/login_server/src/main/java/by/jrr/Application.java

```diff
@@ -0,0 +1,12 @@
+package by.jrr;
+
+import by.jrr.bean.User;
+
+public class Application {
+
+    public static void main(String[] args) {
+        User user = new User("Login", "Password");
+
+        user.LogIn("Login", "Password");
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.